iac/terraform/aws/dynamodb/variable.tf

```diff
@@ -0,0 +1,9 @@
+variable "table_name" {
```

Comment:
Table name should not be a variable. It should be a fix value but name prefix will change with environment.

Reply:
Remove the variable name because it was unused.
You will find this with this [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/17da4268b8fa1b4b6806ef56f93e80823641c2ae)